tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -477,4 +463,23 @@ public function testEnsureDefaultsAreUnescapedFromSchemaIntrospection() : void
         $onlineTable = $this->_sm->listTableDetails("test_column_defaults_with_create");
         self::assertSame($default, $onlineTable->getColumn('col1')->getDefault());
     }
+
+    protected function getEscapedLiterals() : array
+    {
+        return [
+            "\\0", // An ASCII NUL (X'00') character
```

Comment:
This looks like a good candidate for a data provider where comments are array keys.

Reply:
Fixed.